modules/extensions-spring-support/src/test/resources/models/vegan.json

```diff
@@ -1222,6 +1222,11 @@
                 "requiresConnection": true,
                 "requiresConfiguration": true
               },
+              "keysResolver": {
```

Comment:
This is a bug that was fixed. The key resolver is inherited by a metadata scope at extension level.

Reply:
Is it possible to add a test for this fix?